include/sway/config.h

```diff
@@ -285,6 +285,7 @@ struct output_config {
 	int max_render_time; // In milliseconds
 	int adaptive_sync;
 	enum render_bit_depth render_bit_depth;
+	int tearing_allowed;
```

Comment:
How about following the name of the `allow_tearing` option as well?

Reply:
Done.